src/main.js

```diff
@@ -5,170 +5,170 @@ var dateFormating = moment(myDate).format('LL');
 
 console.log(dateFormating);
 
-let destinations = [
+let destinationsList = [
```

Comment:
This file is getting huge. It's a very common practice to split code into different files, like "modules". My recommendation: Create a `scripts/` folder and move this `destinationList` to it.

At the end of the file you must write `export default destinationList` so that you can import it from other files


```js
/src/scripts/destinationList.js

const destinationsList = [ .... ]

export default destinationsList
```

```js
// src/main.js
import moment from "moment";
import destinationList from './scripts/destinationList.js';

...

```

Reply:
This is the error I'm getting
<img width="1792" alt="Screenshot 2020-09-12 at 14 39 25" src="https://user-images.githubusercontent.com/62510606/92995680-cbbca180-f505-11ea-9967-e7536f35813c.png">